[2023-12-23T09:13:33.500+0000] {processor.py:161} INFO - Started process (PID=34) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:13:33.505+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:13:33.516+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:13:33.516+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:13:33.945+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:13:34.351+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:13:34.351+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:13:34.367+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:13:34.367+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:13:34.380+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.902 seconds
[2023-12-23T09:14:05.135+0000] {processor.py:161} INFO - Started process (PID=44) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:05.136+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:14:05.137+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:05.137+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:05.532+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:05.549+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:05.549+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:14:05.561+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:05.561+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:14:05.570+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.437 seconds
[2023-12-23T09:14:13.587+0000] {processor.py:161} INFO - Started process (PID=54) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:13.592+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:14:13.596+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:13.595+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:15.679+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:15.781+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:15.781+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:14:08.998826+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '9cdb804e-91a3-4b56-8c82-716f65e67d4b'}
[2023-12-23T09:14:15.801+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:15.800+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=create_table, execution_date=20231223T091408, start_date=20231223T091410, end_date=20231223T091415
[2023-12-23T09:14:15.809+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.create_table manual__2023-12-23T09:14:08.998826+00:00 [failed]> in state failed
[2023-12-23T09:14:15.861+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:15.860+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:14:15.894+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:15.893+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:14:15.902+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 2.327 seconds
[2023-12-23T09:14:23.844+0000] {processor.py:161} INFO - Started process (PID=56) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:23.846+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:14:23.851+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:23.851+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:24.548+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:24.589+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:24.588+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:14:08.998826+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '4bab415b-5cf7-4f17-a02f-1252ad56f8b5'}
[2023-12-23T09:14:24.602+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:24.602+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=extract_from_file, execution_date=20231223T091408, start_date=20231223T091410, end_date=20231223T091424
[2023-12-23T09:14:24.609+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.extract_from_file manual__2023-12-23T09:14:08.998826+00:00 [failed]> in state failed
[2023-12-23T09:14:24.611+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:24.610+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:14:08.998826+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '4bab415b-5cf7-4f17-a02f-1252ad56f8b5'}
[2023-12-23T09:14:24.614+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:24.614+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=extract_from_file, execution_date=20231223T091408, start_date=20231223T091410, end_date=20231223T091424
[2023-12-23T09:14:24.616+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.extract_from_file manual__2023-12-23T09:14:08.998826+00:00 [failed]> in state failed
[2023-12-23T09:14:24.628+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:24.627+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:14:24.637+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:24.637+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:14:24.647+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.808 seconds
[2023-12-23T09:14:33.953+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:33.954+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:14:33.957+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:33.957+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:34.322+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:14:34.356+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.356+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:14:12.359815+00:00', 'Hostname': '305a147a108b', 'External Executor Id': 'bebd3350-7bd0-42dd-b908-64f9038dfb06'}
[2023-12-23T09:14:34.363+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.363+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=extract_from_file, execution_date=20231223T091412, start_date=20231223T091413, end_date=20231223T091434
[2023-12-23T09:14:34.368+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.extract_from_file manual__2023-12-23T09:14:12.359815+00:00 [failed]> in state failed
[2023-12-23T09:14:34.370+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.370+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:14:12.359815+00:00', 'Hostname': '305a147a108b', 'External Executor Id': 'ffabc6d7-2e78-4205-ba96-f6db35682ca6'}
[2023-12-23T09:14:34.373+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.373+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=create_table, execution_date=20231223T091412, start_date=20231223T091413, end_date=20231223T091434
[2023-12-23T09:14:34.375+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.create_table manual__2023-12-23T09:14:12.359815+00:00 [failed]> in state failed
[2023-12-23T09:14:34.376+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.376+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:14:12.359815+00:00', 'Hostname': '305a147a108b', 'External Executor Id': 'bebd3350-7bd0-42dd-b908-64f9038dfb06'}
[2023-12-23T09:14:34.379+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.379+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=extract_from_file, execution_date=20231223T091412, start_date=20231223T091413, end_date=20231223T091434
[2023-12-23T09:14:34.381+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.extract_from_file manual__2023-12-23T09:14:12.359815+00:00 [failed]> in state failed
[2023-12-23T09:14:34.390+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.390+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:14:34.399+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:14:34.399+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:14:34.409+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.472 seconds
[2023-12-23T09:15:04.793+0000] {processor.py:161} INFO - Started process (PID=73) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:15:04.796+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:15:04.807+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:04.805+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:15:05.615+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:15:05.661+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:05.661+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'extractor_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:14:12.359815+00:00', 'Hostname': '305a147a108b', 'External Executor Id': 'ffabc6d7-2e78-4205-ba96-f6db35682ca6'}
[2023-12-23T09:15:05.670+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:05.670+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extractor_loader_pg, task_id=create_table, execution_date=20231223T091412, start_date=20231223T091413, end_date=20231223T091505
[2023-12-23T09:15:05.677+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: extractor_loader_pg.create_table manual__2023-12-23T09:14:12.359815+00:00 [failed]> in state failed
[2023-12-23T09:15:05.703+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:05.702+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:15:05.715+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:05.715+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:15:05.726+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.952 seconds
[2023-12-23T09:15:36.178+0000] {processor.py:161} INFO - Started process (PID=83) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:15:36.179+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:15:36.181+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:36.180+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:15:36.564+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:15:36.580+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:36.580+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:15:36.592+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:15:36.592+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:15:36.602+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.426 seconds
[2023-12-23T09:16:07.515+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:16:07.516+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:16:07.517+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:16:07.517+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:16:07.863+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:16:07.879+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:16:07.878+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:16:07.889+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:16:07.889+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:16:07.899+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.387 seconds
[2023-12-23T09:16:38.049+0000] {processor.py:161} INFO - Started process (PID=102) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:16:38.051+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:16:38.053+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:16:38.052+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:16:38.392+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:16:38.408+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:16:38.408+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:16:38.420+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:16:38.420+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:16:38.428+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.381 seconds
[2023-12-23T09:17:09.363+0000] {processor.py:161} INFO - Started process (PID=111) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:09.365+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:17:09.368+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:09.368+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:09.959+0000] {processor.py:840} INFO - DAG(s) 'extractor_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:10.054+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:10.053+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:17:10.085+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:10.085+0000] {dag.py:3820} INFO - Setting next_dagrun for extractor_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:17:10.100+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.741 seconds
[2023-12-23T09:17:26.617+0000] {processor.py:161} INFO - Started process (PID=119) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:26.619+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:17:26.646+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:26.643+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:27.142+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:27.234+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:27.234+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:data_loader_pg
[2023-12-23T09:17:27.242+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:27.242+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:data_loader_pg
[2023-12-23T09:17:27.247+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:27.247+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:data_loader_pg
[2023-12-23T09:17:27.248+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:27.248+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:17:27.266+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:27.264+0000] {dag.py:3055} INFO - Creating ORM DAG for data_loader_pg
[2023-12-23T09:17:27.278+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:27.278+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:17:27.289+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.678 seconds
[2023-12-23T09:17:57.927+0000] {processor.py:161} INFO - Started process (PID=128) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:57.930+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:17:57.933+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:57.933+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:58.362+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:17:58.385+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:58.385+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:17:58.399+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:17:58.399+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:17:58.408+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.485 seconds
[2023-12-23T09:18:29.303+0000] {processor.py:161} INFO - Started process (PID=137) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:18:29.304+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:18:29.308+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:18:29.308+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:18:29.693+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:18:29.713+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:18:29.713+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:18:29.734+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:18:29.734+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:18:29.744+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.446 seconds
[2023-12-23T09:19:00.666+0000] {processor.py:161} INFO - Started process (PID=146) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:19:00.668+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:19:00.669+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:19:00.669+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:19:01.075+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:19:01.095+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:19:01.094+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:19:01.107+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:19:01.107+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:19:01.117+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.459 seconds
[2023-12-23T09:19:31.335+0000] {processor.py:161} INFO - Started process (PID=155) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:19:31.338+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:19:31.339+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:19:31.338+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:19:31.707+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:19:31.724+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:19:31.724+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:19:31.735+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:19:31.735+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:19:31.745+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.414 seconds
[2023-12-23T09:20:02.692+0000] {processor.py:161} INFO - Started process (PID=164) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:20:02.696+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:20:02.697+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:20:02.696+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:20:03.154+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:20:03.193+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:20:03.193+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:20:03.206+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:20:03.206+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:20:03.216+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.545 seconds
[2023-12-23T09:20:34.262+0000] {processor.py:161} INFO - Started process (PID=173) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:20:34.264+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:20:34.265+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:20:34.265+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:20:34.700+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:20:34.720+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:20:34.720+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:20:34.732+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:20:34.732+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:20:34.741+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.486 seconds
[2023-12-23T09:21:05.659+0000] {processor.py:161} INFO - Started process (PID=182) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:05.662+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:21:05.664+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:05.663+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:06.298+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:06.321+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:06.321+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:21:06.334+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:06.334+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:21:06.344+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.694 seconds
[2023-12-23T09:21:35.944+0000] {processor.py:161} INFO - Started process (PID=194) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:35.947+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:21:35.948+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:35.947+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:36.408+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:36.467+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:36.466+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:21:27.013827+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '51adb9dc-9506-4e01-a7cf-6e0f2a7a8c86'}
[2023-12-23T09:21:36.474+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:36.474+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=extract_from_file, execution_date=20231223T092127, start_date=20231223T092128, end_date=20231223T092136
[2023-12-23T09:21:36.481+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.extract_from_file manual__2023-12-23T09:21:27.013827+00:00 [failed]> in state failed
[2023-12-23T09:21:36.493+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:36.493+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:21:36.503+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:36.503+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:21:36.512+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.574 seconds
[2023-12-23T09:21:45.947+0000] {processor.py:161} INFO - Started process (PID=197) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:45.948+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:21:45.949+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:45.949+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:46.186+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:21:46.200+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:46.199+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:21:27.013827+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '32633e1a-a69a-40e9-92b3-05a1d4641357'}
[2023-12-23T09:21:46.206+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:46.206+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=create_table, execution_date=20231223T092127, start_date=20231223T092128, end_date=20231223T092146
[2023-12-23T09:21:46.210+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.create_table manual__2023-12-23T09:21:27.013827+00:00 [failed]> in state failed
[2023-12-23T09:21:46.211+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:46.211+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:21:27.013827+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '32633e1a-a69a-40e9-92b3-05a1d4641357'}
[2023-12-23T09:21:46.215+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:46.215+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=create_table, execution_date=20231223T092127, start_date=20231223T092128, end_date=20231223T092146
[2023-12-23T09:21:46.217+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.create_table manual__2023-12-23T09:21:27.013827+00:00 [failed]> in state failed
[2023-12-23T09:21:46.225+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:46.225+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:21:46.234+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:21:46.234+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:21:46.242+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.297 seconds
[2023-12-23T09:22:16.515+0000] {processor.py:161} INFO - Started process (PID=206) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:22:16.517+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:22:16.525+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:22:16.519+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:22:17.120+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:22:17.157+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:22:17.157+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:22:17.170+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:22:17.170+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:22:17.181+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.671 seconds
[2023-12-23T09:22:47.819+0000] {processor.py:161} INFO - Started process (PID=215) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:22:47.821+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:22:47.822+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:22:47.821+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:22:48.302+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:22:48.320+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:22:48.320+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:22:48.332+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:22:48.332+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:22:48.341+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.526 seconds
[2023-12-23T09:23:18.470+0000] {processor.py:161} INFO - Started process (PID=224) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:18.474+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:23:18.476+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:18.475+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:19.091+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:19.149+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:19.148+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:23:19.170+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:19.169+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:23:19.189+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.727 seconds
[2023-12-23T09:23:49.820+0000] {processor.py:161} INFO - Started process (PID=233) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:49.822+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:23:49.822+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:49.822+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:50.289+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:50.307+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:50.307+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:23:50.320+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:50.319+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:23:50.331+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.515 seconds
[2023-12-23T09:23:56.514+0000] {processor.py:161} INFO - Started process (PID=239) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:56.515+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:23:56.516+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:56.516+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:56.878+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:23:56.895+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:56.894+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:23:51.420103+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '0eed2b50-f072-4ae5-b02c-4c5e1af0205d'}
[2023-12-23T09:23:56.903+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:56.903+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=create_table, execution_date=20231223T092351, start_date=20231223T092352, end_date=20231223T092356
[2023-12-23T09:23:56.909+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.create_table manual__2023-12-23T09:23:51.420103+00:00 [failed]> in state failed
[2023-12-23T09:23:56.923+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:56.923+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:23:56.934+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:23:56.934+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:23:56.945+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.436 seconds
[2023-12-23T09:24:06.679+0000] {processor.py:161} INFO - Started process (PID=251) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:06.681+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:24:06.682+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:06.681+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:07.194+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:07.214+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:07.213+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:23:51.420103+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '385defed-c670-4a8d-906a-87fa414e2c50'}
[2023-12-23T09:24:07.221+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:07.221+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=extract_from_file, execution_date=20231223T092351, start_date=20231223T092352, end_date=20231223T092407
[2023-12-23T09:24:07.226+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.extract_from_file manual__2023-12-23T09:23:51.420103+00:00 [failed]> in state failed
[2023-12-23T09:24:07.228+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:07.227+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:23:51.420103+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '385defed-c670-4a8d-906a-87fa414e2c50'}
[2023-12-23T09:24:07.231+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:07.231+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=extract_from_file, execution_date=20231223T092351, start_date=20231223T092352, end_date=20231223T092407
[2023-12-23T09:24:07.233+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.extract_from_file manual__2023-12-23T09:23:51.420103+00:00 [failed]> in state failed
[2023-12-23T09:24:07.243+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:07.242+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:24:07.251+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:07.251+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:24:07.261+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.588 seconds
[2023-12-23T09:24:16.788+0000] {processor.py:161} INFO - Started process (PID=254) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:16.790+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:24:16.794+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:16.793+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:17.286+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:17.307+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.307+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:23:57.679251+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '9c6c8885-3c36-4ac9-b5c9-d4be9491ad57'}
[2023-12-23T09:24:17.316+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.316+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=extract_from_file, execution_date=20231223T092357, start_date=20231223T092359, end_date=20231223T092417
[2023-12-23T09:24:17.325+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.extract_from_file manual__2023-12-23T09:23:57.679251+00:00 [failed]> in state failed
[2023-12-23T09:24:17.329+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.329+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:23:57.679251+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '03f1e4b1-0e91-44ff-8dc1-919b2f825d3e'}
[2023-12-23T09:24:17.334+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.334+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=create_table, execution_date=20231223T092357, start_date=20231223T092359, end_date=20231223T092417
[2023-12-23T09:24:17.336+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.create_table manual__2023-12-23T09:23:57.679251+00:00 [failed]> in state failed
[2023-12-23T09:24:17.337+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.337+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'extract_from_file', 'Run Id': 'manual__2023-12-23T09:23:57.679251+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '9c6c8885-3c36-4ac9-b5c9-d4be9491ad57'}
[2023-12-23T09:24:17.345+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.344+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=extract_from_file, execution_date=20231223T092357, start_date=20231223T092359, end_date=20231223T092417
[2023-12-23T09:24:17.347+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.extract_from_file manual__2023-12-23T09:23:57.679251+00:00 [failed]> in state failed
[2023-12-23T09:24:17.357+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.357+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:24:17.367+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:17.367+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:24:17.376+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.594 seconds
[2023-12-23T09:24:47.745+0000] {processor.py:161} INFO - Started process (PID=263) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:47.748+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:24:47.749+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:47.749+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:48.155+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:24:48.187+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:48.187+0000] {taskinstance.py:2701} ERROR - {'DAG Id': 'data_loader_pg', 'Task Id': 'create_table', 'Run Id': 'manual__2023-12-23T09:23:57.679251+00:00', 'Hostname': '305a147a108b', 'External Executor Id': '03f1e4b1-0e91-44ff-8dc1-919b2f825d3e'}
[2023-12-23T09:24:48.198+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:48.198+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=data_loader_pg, task_id=create_table, execution_date=20231223T092357, start_date=20231223T092359, end_date=20231223T092448
[2023-12-23T09:24:48.203+0000] {processor.py:791} INFO - Executed failure callback for <TaskInstance: data_loader_pg.create_table manual__2023-12-23T09:23:57.679251+00:00 [failed]> in state failed
[2023-12-23T09:24:48.212+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:48.211+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:24:48.221+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:24:48.221+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:24:48.228+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.488 seconds
[2023-12-23T09:25:18.307+0000] {processor.py:161} INFO - Started process (PID=272) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:25:18.309+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:25:18.310+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:25:18.310+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:25:18.727+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:25:18.749+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:25:18.749+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:25:18.761+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:25:18.761+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:25:18.771+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.470 seconds
[2023-12-23T09:25:49.725+0000] {processor.py:161} INFO - Started process (PID=281) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:25:49.727+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:25:49.727+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:25:49.727+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:25:50.067+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:25:50.081+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:25:50.081+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:25:50.092+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:25:50.092+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:25:50.106+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.383 seconds
[2023-12-23T09:26:20.203+0000] {processor.py:161} INFO - Started process (PID=290) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:26:20.205+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:26:20.206+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:26:20.205+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:26:20.716+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:26:20.741+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:26:20.740+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:26:20.753+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:26:20.753+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:26:20.765+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.565 seconds
[2023-12-23T09:26:51.843+0000] {processor.py:161} INFO - Started process (PID=299) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:26:51.845+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:26:51.846+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:26:51.846+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:26:52.312+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:26:52.329+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:26:52.329+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:26:52.352+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:26:52.352+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:26:52.361+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.526 seconds
[2023-12-23T09:27:23.257+0000] {processor.py:161} INFO - Started process (PID=308) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:27:23.259+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:27:23.260+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:27:23.259+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:27:23.690+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:27:23.711+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:27:23.711+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:27:23.725+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:27:23.725+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:27:23.736+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.485 seconds
[2023-12-23T09:27:54.662+0000] {processor.py:161} INFO - Started process (PID=317) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:27:54.663+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:27:54.663+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:27:54.663+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:27:54.989+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:27:55.004+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:27:55.004+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:27:55.014+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:27:55.014+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:27:55.022+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.364 seconds
[2023-12-23T09:28:25.188+0000] {processor.py:161} INFO - Started process (PID=326) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:28:25.190+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:28:25.191+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:28:25.191+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:28:25.597+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:28:25.615+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:28:25.614+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:28:25.627+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:28:25.627+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:28:25.637+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.453 seconds
[2023-12-23T09:28:52.513+0000] {processor.py:161} INFO - Started process (PID=335) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:28:52.516+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:28:52.517+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:28:52.516+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:28:53.080+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:28:53.129+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:28:53.129+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:28:53.148+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:28:53.148+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:28:53.176+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.684 seconds
[2023-12-23T09:29:23.969+0000] {processor.py:161} INFO - Started process (PID=344) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:29:23.972+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:29:23.974+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:29:23.973+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:29:24.405+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:29:24.427+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:29:24.427+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:29:24.450+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:29:24.450+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:29:24.461+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.511 seconds
[2023-12-23T09:29:54.533+0000] {processor.py:161} INFO - Started process (PID=353) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:29:54.534+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:29:54.535+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:29:54.535+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:29:54.887+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:29:54.903+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:29:54.902+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:29:54.914+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:29:54.914+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:29:54.922+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.392 seconds
[2023-12-23T09:30:25.024+0000] {processor.py:161} INFO - Started process (PID=362) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:30:25.025+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:30:25.026+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:30:25.026+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:30:25.376+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:30:25.394+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:30:25.394+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:30:25.405+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:30:25.405+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:30:25.413+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.393 seconds
[2023-12-23T09:30:56.191+0000] {processor.py:161} INFO - Started process (PID=371) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:30:56.193+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:30:56.194+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:30:56.194+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:30:56.574+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:30:56.590+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:30:56.590+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:30:56.601+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:30:56.601+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:30:56.610+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.424 seconds
[2023-12-23T09:31:26.653+0000] {processor.py:161} INFO - Started process (PID=380) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:31:26.654+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:31:26.655+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:31:26.654+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:31:26.993+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:31:27.010+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:31:27.010+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:31:27.020+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:31:27.020+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:31:27.031+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.381 seconds
[2023-12-23T09:31:57.300+0000] {processor.py:161} INFO - Started process (PID=389) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:31:57.302+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:31:57.304+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:31:57.303+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:31:57.675+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:31:57.674+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 13, in <module>
    from tableManger import db_util
ImportError: cannot import name 'db_util' from 'tableManger' (unknown location)
[2023-12-23T09:31:57.676+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:31:57.696+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.402 seconds
[2023-12-23T09:32:19.646+0000] {processor.py:161} INFO - Started process (PID=396) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:19.648+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:32:19.652+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:19.650+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:20.130+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:20.152+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:20.151+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:32:20.165+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:20.165+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:32:20.178+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.556 seconds
[2023-12-23T09:32:27.958+0000] {processor.py:161} INFO - Started process (PID=399) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:27.959+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:32:27.961+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:27.960+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:28.382+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:28.381+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 13, in <module>
    from tableManger import db_util
ImportError: cannot import name 'db_util' from 'tableManger' (unknown location)
[2023-12-23T09:32:28.383+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:28.396+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.443 seconds
[2023-12-23T09:32:44.499+0000] {processor.py:161} INFO - Started process (PID=407) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:44.501+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:32:44.503+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:44.502+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:44.985+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:32:45.013+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:45.013+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:32:45.035+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:32:45.035+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:32:45.050+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.561 seconds
[2023-12-23T09:33:15.732+0000] {processor.py:161} INFO - Started process (PID=416) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:33:15.734+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:33:15.736+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:33:15.735+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:33:16.222+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:33:16.246+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:33:16.246+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:33:16.259+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:33:16.259+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:33:16.269+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.545 seconds
[2023-12-23T09:33:46.873+0000] {processor.py:161} INFO - Started process (PID=425) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T09:33:46.875+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T09:33:46.877+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:33:46.876+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T09:33:47.289+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T09:33:47.309+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:33:47.309+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T09:33:47.322+0000] {logging_mixin.py:188} INFO - [2023-12-23T09:33:47.322+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T09:33:47.332+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.467 seconds
[2023-12-23T10:15:45.920+0000] {processor.py:161} INFO - Started process (PID=34) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:15:45.922+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:15:45.925+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:15:45.925+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:15:46.288+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:15:46.287+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:15:46.289+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:15:46.322+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.408 seconds
[2023-12-23T10:16:16.544+0000] {processor.py:161} INFO - Started process (PID=43) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:16:16.546+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:16:16.564+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:16:16.563+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:16:17.131+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:16:17.130+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:16:17.132+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:16:17.151+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.613 seconds
[2023-12-23T10:16:47.266+0000] {processor.py:161} INFO - Started process (PID=53) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:16:47.267+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:16:47.267+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:16:47.267+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:16:47.675+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:16:47.674+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:16:47.675+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:16:47.686+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.423 seconds
[2023-12-23T10:17:18.558+0000] {processor.py:161} INFO - Started process (PID=62) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:17:18.559+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:17:18.559+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:17:18.559+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:17:18.894+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:17:18.893+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:17:18.894+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:17:18.905+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.349 seconds
[2023-12-23T10:17:48.969+0000] {processor.py:161} INFO - Started process (PID=71) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:17:48.970+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:17:48.971+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:17:48.971+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:17:49.311+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:17:49.311+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:17:49.312+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:17:49.323+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.356 seconds
[2023-12-23T10:18:19.370+0000] {processor.py:161} INFO - Started process (PID=80) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:18:19.371+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:18:19.371+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:18:19.371+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:18:19.718+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:18:19.718+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:18:19.719+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:18:19.729+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.362 seconds
[2023-12-23T10:18:49.924+0000] {processor.py:161} INFO - Started process (PID=89) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:18:49.925+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:18:49.926+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:18:49.925+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:18:50.488+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:18:50.487+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:18:50.488+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:18:50.508+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.588 seconds
[2023-12-23T10:19:21.262+0000] {processor.py:161} INFO - Started process (PID=98) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:19:21.263+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:19:21.264+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:19:21.264+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:19:21.738+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:19:21.737+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:19:21.738+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:19:21.754+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.496 seconds
[2023-12-23T10:19:51.781+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:19:51.782+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:19:51.783+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:19:51.783+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:19:52.833+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:19:52.831+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:19:52.833+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:19:52.885+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 1.106 seconds
[2023-12-23T10:20:23.257+0000] {processor.py:161} INFO - Started process (PID=116) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:20:23.258+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:20:23.259+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:20:23.259+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:20:23.650+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:20:23.649+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:20:23.650+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:20:23.662+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.408 seconds
[2023-12-23T10:20:53.773+0000] {processor.py:161} INFO - Started process (PID=125) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:20:53.775+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:20:53.777+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:20:53.776+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:20:54.345+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:20:54.344+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:20:54.345+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:20:54.361+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.596 seconds
[2023-12-23T10:21:25.312+0000] {processor.py:161} INFO - Started process (PID=134) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:21:25.313+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:21:25.314+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:21:25.314+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:21:25.710+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:21:25.709+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:21:25.710+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:21:25.723+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.414 seconds
[2023-12-23T10:21:55.796+0000] {processor.py:161} INFO - Started process (PID=143) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:21:55.797+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:21:55.798+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:21:55.798+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:21:56.195+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:21:56.194+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:21:56.196+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:21:56.207+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.416 seconds
[2023-12-23T10:22:27.122+0000] {processor.py:161} INFO - Started process (PID=151) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:22:27.125+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:22:27.125+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:22:27.125+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:22:27.518+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:22:27.517+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:22:27.518+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:22:27.532+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.415 seconds
[2023-12-23T10:22:57.888+0000] {processor.py:161} INFO - Started process (PID=160) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:22:57.890+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:22:57.891+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:22:57.891+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:22:58.343+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:22:58.343+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:22:58.344+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:22:58.368+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.486 seconds
[2023-12-23T10:23:29.180+0000] {processor.py:161} INFO - Started process (PID=169) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:23:29.183+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:23:29.184+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:23:29.184+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:23:29.602+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:23:29.601+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:23:29.602+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:23:29.615+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.440 seconds
[2023-12-23T10:23:59.728+0000] {processor.py:161} INFO - Started process (PID=178) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:23:59.729+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:23:59.729+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:23:59.729+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:24:00.150+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:24:00.149+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:24:00.150+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:24:00.166+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.441 seconds
[2023-12-23T10:24:31.031+0000] {processor.py:161} INFO - Started process (PID=187) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:24:31.033+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:24:31.034+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:24:31.034+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:24:31.452+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:24:31.451+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:24:31.452+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:24:31.462+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.437 seconds
[2023-12-23T10:25:01.627+0000] {processor.py:161} INFO - Started process (PID=196) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:25:01.630+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:25:01.633+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:25:01.632+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:25:02.084+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:25:02.083+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:25:02.084+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:25:02.101+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.498 seconds
[2023-12-23T10:25:32.174+0000] {processor.py:161} INFO - Started process (PID=205) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:25:32.176+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:25:32.177+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:25:32.177+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:25:32.673+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:25:32.672+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:25:32.674+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:25:32.710+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.538 seconds
[2023-12-23T10:26:02.953+0000] {processor.py:161} INFO - Started process (PID=214) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:26:02.954+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:26:02.955+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:26:02.955+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:26:03.350+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:26:03.349+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:26:03.350+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:26:03.363+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.412 seconds
[2023-12-23T10:26:33.432+0000] {processor.py:161} INFO - Started process (PID=223) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:26:33.434+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:26:33.435+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:26:33.435+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:26:33.836+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:26:33.835+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:26:33.837+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:26:33.850+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.423 seconds
[2023-12-23T10:27:04.829+0000] {processor.py:161} INFO - Started process (PID=232) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:27:04.832+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:27:04.834+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:27:04.834+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:27:05.283+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:27:05.282+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:27:05.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:27:05.298+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.477 seconds
[2023-12-23T10:27:36.203+0000] {processor.py:161} INFO - Started process (PID=241) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:27:36.204+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:27:36.204+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:27:36.204+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:27:36.614+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:27:36.613+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:27:36.614+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:27:36.629+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.430 seconds
[2023-12-23T10:28:07.618+0000] {processor.py:161} INFO - Started process (PID=250) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:28:07.620+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:28:07.621+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:28:07.620+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:28:07.956+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:28:07.955+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:28:07.956+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:28:07.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.360 seconds
[2023-12-23T10:28:38.247+0000] {processor.py:161} INFO - Started process (PID=259) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:28:38.248+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:28:38.249+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:28:38.249+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:28:38.643+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:28:38.642+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:28:38.643+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:28:38.656+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.411 seconds
[2023-12-23T10:29:09.628+0000] {processor.py:161} INFO - Started process (PID=268) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:29:09.631+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:29:09.632+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:29:09.632+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:29:10.172+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:29:10.171+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:29:10.172+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:29:10.195+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.573 seconds
[2023-12-23T10:29:40.977+0000] {processor.py:161} INFO - Started process (PID=282) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:29:40.979+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:29:40.980+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:29:40.979+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:29:41.334+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:29:41.333+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:29:41.334+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:29:41.346+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.373 seconds
[2023-12-23T10:30:11.427+0000] {processor.py:161} INFO - Started process (PID=291) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:30:11.430+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:30:11.432+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:30:11.432+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:30:11.747+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:30:11.746+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:30:11.747+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:30:11.759+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.337 seconds
[2023-12-23T10:30:41.809+0000] {processor.py:161} INFO - Started process (PID=300) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:30:41.812+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:30:41.813+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:30:41.813+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:30:42.282+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:30:42.281+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:30:42.282+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:30:42.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.535 seconds
[2023-12-23T10:31:13.334+0000] {processor.py:161} INFO - Started process (PID=309) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:31:13.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:31:13.338+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:31:13.337+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:31:14.200+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:31:14.197+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:31:14.201+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:31:14.246+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.932 seconds
[2023-12-23T10:31:44.647+0000] {processor.py:161} INFO - Started process (PID=318) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:31:44.649+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:31:44.650+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:31:44.649+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:31:45.184+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:31:45.182+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:31:45.184+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:31:45.200+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.556 seconds
[2023-12-23T10:32:15.969+0000] {processor.py:161} INFO - Started process (PID=327) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:32:15.970+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:32:15.971+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:32:15.971+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:32:16.362+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:32:16.361+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:32:16.363+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:32:16.377+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.416 seconds
[2023-12-23T10:32:47.289+0000] {processor.py:161} INFO - Started process (PID=336) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:32:47.292+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:32:47.293+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:32:47.292+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:32:47.550+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:32:47.550+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:32:47.551+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:32:47.562+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.279 seconds
[2023-12-23T10:33:17.653+0000] {processor.py:161} INFO - Started process (PID=345) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:33:17.656+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:33:17.657+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:33:17.657+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:33:18.115+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:33:18.115+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:33:18.116+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:33:18.134+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.505 seconds
[2023-12-23T10:33:48.908+0000] {processor.py:161} INFO - Started process (PID=354) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:33:48.911+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:33:48.912+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:33:48.912+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:33:49.421+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:33:49.420+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:33:49.421+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:33:49.438+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.537 seconds
[2023-12-23T10:34:20.266+0000] {processor.py:161} INFO - Started process (PID=363) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:34:20.269+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:34:20.270+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:34:20.269+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:34:20.733+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:34:20.732+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:34:20.733+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:34:20.750+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.493 seconds
[2023-12-23T10:34:51.623+0000] {processor.py:161} INFO - Started process (PID=371) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:34:51.624+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:34:51.625+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:34:51.624+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:34:51.991+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:34:51.990+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:34:51.991+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:34:52.004+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.384 seconds
[2023-12-23T10:35:22.102+0000] {processor.py:161} INFO - Started process (PID=380) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:35:22.104+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:35:22.106+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:35:22.105+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:35:22.560+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:35:22.559+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:35:22.560+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:35:22.578+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.486 seconds
[2023-12-23T10:35:53.454+0000] {processor.py:161} INFO - Started process (PID=389) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:35:53.455+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:35:53.456+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:35:53.456+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:35:53.887+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:35:53.886+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:35:53.888+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:35:53.903+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.456 seconds
[2023-12-23T10:36:24.010+0000] {processor.py:161} INFO - Started process (PID=398) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:36:24.012+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:36:24.014+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:36:24.013+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:36:24.502+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:36:24.501+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:36:24.503+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:36:24.529+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.526 seconds
[2023-12-23T10:36:55.358+0000] {processor.py:161} INFO - Started process (PID=407) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:36:55.360+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:36:55.361+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:36:55.361+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:36:55.806+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:36:55.805+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:36:55.806+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:36:55.821+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.468 seconds
[2023-12-23T10:37:26.422+0000] {processor.py:161} INFO - Started process (PID=416) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:37:26.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:37:26.426+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:37:26.425+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:37:26.869+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:37:26.868+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:37:26.869+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:37:26.884+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.469 seconds
[2023-12-23T10:37:57.225+0000] {processor.py:161} INFO - Started process (PID=425) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:37:57.227+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:37:57.229+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:37:57.228+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:37:57.637+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:37:57.636+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:37:57.637+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:37:57.652+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.431 seconds
[2023-12-23T10:38:28.575+0000] {processor.py:161} INFO - Started process (PID=434) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:38:28.580+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:38:28.582+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:38:28.581+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:38:29.059+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:38:29.058+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/data_loader.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_loader.py", line 73, in <module>
    create_tables = PythonOperator(task_id="create_table", python_callable=create_table)
NameError: name 'create_table' is not defined
[2023-12-23T10:38:29.059+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:38:29.075+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.508 seconds
[2023-12-23T10:38:39.743+0000] {processor.py:161} INFO - Started process (PID=437) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:38:39.769+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:38:39.773+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:38:39.773+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:38:40.270+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:38:40.363+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:38:40.363+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:38:40.377+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:38:40.376+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:38:40.402+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.662 seconds
[2023-12-23T10:39:11.134+0000] {processor.py:161} INFO - Started process (PID=446) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:39:11.136+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:39:11.137+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:39:11.136+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:39:11.772+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:39:11.793+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:39:11.793+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:39:11.808+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:39:11.808+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:39:11.819+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.688 seconds
[2023-12-23T10:39:42.553+0000] {processor.py:161} INFO - Started process (PID=461) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:39:42.555+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:39:42.556+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:39:42.556+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:39:43.117+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:39:43.174+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:39:43.174+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:39:43.192+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:39:43.192+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:39:43.203+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.655 seconds
[2023-12-23T10:40:14.035+0000] {processor.py:161} INFO - Started process (PID=470) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:40:14.039+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:40:14.041+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:40:14.040+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:40:14.590+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:40:14.619+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:40:14.618+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:40:14.631+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:40:14.631+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:40:14.641+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.612 seconds
[2023-12-23T10:40:45.450+0000] {processor.py:161} INFO - Started process (PID=479) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:40:45.451+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:40:45.452+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:40:45.451+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:40:45.774+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:40:45.793+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:40:45.793+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:40:45.805+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:40:45.805+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:40:45.815+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.370 seconds
[2023-12-23T10:41:15.965+0000] {processor.py:161} INFO - Started process (PID=488) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:41:15.969+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:41:15.970+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:41:15.969+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:41:16.430+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:41:16.453+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:41:16.453+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:41:16.465+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:41:16.465+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:41:16.474+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.518 seconds
[2023-12-23T10:41:47.364+0000] {processor.py:161} INFO - Started process (PID=497) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:41:47.366+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:41:47.366+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:41:47.366+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:41:47.759+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:41:47.780+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:41:47.780+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:41:47.793+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:41:47.793+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:41:47.803+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.442 seconds
[2023-12-23T10:42:17.942+0000] {processor.py:161} INFO - Started process (PID=506) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:42:17.946+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:42:17.953+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:42:17.949+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:42:18.386+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:42:18.410+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:42:18.410+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:42:18.422+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:42:18.422+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:42:18.432+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.495 seconds
[2023-12-23T10:42:49.294+0000] {processor.py:161} INFO - Started process (PID=515) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:42:49.295+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:42:49.296+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:42:49.296+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:42:49.708+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:42:49.729+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:42:49.729+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:42:49.743+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:42:49.742+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:42:49.753+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.462 seconds
[2023-12-23T10:43:19.859+0000] {processor.py:161} INFO - Started process (PID=524) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:43:19.863+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:43:19.877+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:43:19.875+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:43:20.289+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:43:20.309+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:43:20.309+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:43:20.321+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:43:20.321+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:43:20.336+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.485 seconds
[2023-12-23T10:43:50.421+0000] {processor.py:161} INFO - Started process (PID=533) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:43:50.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:43:50.425+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:43:50.425+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:43:50.781+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:43:50.797+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:43:50.797+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:43:50.809+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:43:50.809+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:43:50.818+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.405 seconds
[2023-12-23T10:44:21.775+0000] {processor.py:161} INFO - Started process (PID=542) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:44:21.777+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:44:21.778+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:44:21.778+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:44:22.201+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:44:22.223+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:44:22.223+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:44:22.237+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:44:22.237+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:44:22.249+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.480 seconds
[2023-12-23T10:44:53.129+0000] {processor.py:161} INFO - Started process (PID=551) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:44:53.131+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:44:53.132+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:44:53.131+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:44:53.492+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:44:53.511+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:44:53.511+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:44:53.523+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:44:53.523+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:44:53.533+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.409 seconds
[2023-12-23T10:45:23.639+0000] {processor.py:161} INFO - Started process (PID=560) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:45:23.644+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:45:23.646+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:45:23.645+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:45:24.078+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:45:24.098+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:45:24.097+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:45:24.109+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:45:24.108+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:45:24.119+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.487 seconds
[2023-12-23T10:45:55.110+0000] {processor.py:161} INFO - Started process (PID=569) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:45:55.112+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:45:55.112+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:45:55.112+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:45:55.485+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:45:55.502+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:45:55.502+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:45:55.513+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:45:55.513+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:45:55.522+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.416 seconds
[2023-12-23T10:46:26.460+0000] {processor.py:161} INFO - Started process (PID=578) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:46:26.463+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:46:26.464+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:46:26.463+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:46:26.880+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:46:26.898+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:46:26.898+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:46:26.911+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:46:26.911+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:46:26.920+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.467 seconds
[2023-12-23T10:46:57.852+0000] {processor.py:161} INFO - Started process (PID=587) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:46:57.853+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:46:57.855+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:46:57.854+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:46:58.210+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:46:58.227+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:46:58.226+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:46:58.237+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:46:58.237+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:46:58.246+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.399 seconds
[2023-12-23T10:47:29.184+0000] {processor.py:161} INFO - Started process (PID=596) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:47:29.187+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:47:29.188+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:47:29.187+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:47:29.623+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:47:29.644+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:47:29.644+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:47:29.656+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:47:29.656+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:47:29.666+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.489 seconds
[2023-12-23T10:48:00.520+0000] {processor.py:161} INFO - Started process (PID=605) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:48:00.522+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:48:00.522+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:48:00.522+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:48:00.850+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:48:00.865+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:48:00.865+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:48:00.876+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:48:00.875+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:48:00.884+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.369 seconds
[2023-12-23T10:48:31.151+0000] {processor.py:161} INFO - Started process (PID=613) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:48:31.153+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:48:31.154+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:48:31.154+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:48:31.558+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:48:31.579+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:48:31.578+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:48:31.591+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:48:31.590+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:48:31.601+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.455 seconds
[2023-12-23T10:49:01.754+0000] {processor.py:161} INFO - Started process (PID=622) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:49:01.756+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:49:01.757+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:49:01.757+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:49:02.104+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:49:02.121+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:49:02.121+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:49:02.133+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:49:02.133+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:49:02.143+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.392 seconds
[2023-12-23T10:49:32.286+0000] {processor.py:161} INFO - Started process (PID=631) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:49:32.288+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:49:32.290+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:49:32.289+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:49:33.203+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:49:33.260+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:49:33.260+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:49:33.277+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:49:33.277+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:49:33.301+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 1.019 seconds
[2023-12-23T10:50:03.601+0000] {processor.py:161} INFO - Started process (PID=640) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:50:03.602+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:50:03.603+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:50:03.603+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:50:03.930+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:50:03.946+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:50:03.945+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:50:03.957+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:50:03.957+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:50:03.965+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.369 seconds
[2023-12-23T10:50:34.069+0000] {processor.py:161} INFO - Started process (PID=649) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:50:34.071+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:50:34.072+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:50:34.072+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:50:34.442+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:50:34.469+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:50:34.469+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:50:34.497+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:50:34.496+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:50:34.511+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.447 seconds
[2023-12-23T10:51:05.510+0000] {processor.py:161} INFO - Started process (PID=658) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:51:05.516+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:51:05.521+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:51:05.519+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:51:05.993+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:51:06.016+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:51:06.016+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:51:06.029+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:51:06.029+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:51:06.040+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.541 seconds
[2023-12-23T10:51:36.991+0000] {processor.py:161} INFO - Started process (PID=667) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:51:36.993+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:51:36.995+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:51:36.994+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:51:38.014+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:51:38.145+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:51:38.145+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:51:38.182+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:51:38.182+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:51:38.211+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 1.229 seconds
[2023-12-23T10:52:08.324+0000] {processor.py:161} INFO - Started process (PID=676) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:52:08.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:52:08.330+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:52:08.329+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:52:08.745+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:52:08.765+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:52:08.765+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:52:08.778+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:52:08.778+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:52:08.787+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.469 seconds
[2023-12-23T10:52:39.758+0000] {processor.py:161} INFO - Started process (PID=685) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:52:39.759+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:52:39.760+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:52:39.760+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:52:40.164+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:52:40.185+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:52:40.185+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:52:40.197+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:52:40.197+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:52:40.207+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.456 seconds
[2023-12-23T10:53:11.132+0000] {processor.py:161} INFO - Started process (PID=694) to work on /opt/airflow/dags/data_loader.py
[2023-12-23T10:53:11.134+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/data_loader.py for tasks to queue
[2023-12-23T10:53:11.135+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:53:11.135+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/data_loader.py
[2023-12-23T10:53:11.498+0000] {processor.py:840} INFO - DAG(s) 'data_loader_pg' retrieved from /opt/airflow/dags/data_loader.py
[2023-12-23T10:53:11.515+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:53:11.514+0000] {dag.py:3033} INFO - Sync 1 DAGs
[2023-12-23T10:53:11.525+0000] {logging_mixin.py:188} INFO - [2023-12-23T10:53:11.525+0000] {dag.py:3820} INFO - Setting next_dagrun for data_loader_pg to 2023-12-23T00:00:00+00:00, run_after=2023-12-24T00:00:00+00:00
[2023-12-23T10:53:11.534+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/data_loader.py took 0.409 seconds
